game-headed/build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
     implementation project(':java-extras')
```

Comment:
A couple of lines above there's an if check for Java 11+ versions, this can probably be removed entirely before merging.

Reply:
And if the if check is always true, the plugin can be applied in line 5 already (by removing `apply false`)

Reply:
But other than that, this looks good 👍